packages/app-security/src/admin/plugins/secureRouteError.tsx

```diff
@@ -9,10 +18,13 @@ const plugin: SecureRouteErrorPlugin = {
     name: "secure-route-error",
     render() {
         return (
-            <>
+            <div className={notAuthCss}>    
                 <Helmet title={"Not authorized"} />
-                <span>You are not authorized to access this route.</span>
-            </>
+                <img src={authErrorImg} alt="Not Authorized" width="40%"/>
+                <p>You are not authorized to view this route.</p>
```

Comment:
Please use `Typography` component from `@webiny/ui/Typography` package instead. For example,

```js
<Typography use={"body1"} className={...}>               
     You are not authorized to access this route.
 </Typography>
```